pyproject.toml

```diff
@@ -47,12 +47,13 @@ dev = [
     "black",
     "bandit",
     "flake8",
+    "pylint",
     "pytest",
     "pytest-cov"
 ]
 
 [tool.setuptools]
-packages = ["test", "vdb", "vdb.lib"]
+packages = ["vdb", "vdb.lib", "vdb.lib.cve_model"]
```

Comment:
@prabhu Is there a reason we include test in the packages? There isn't any need to include our tests in the distribution that I can think of, so I removed.

Reply:
I usually include tests since they are like docs too.